libsqlite3-sys/libsql/bindgen_bundled_version.rs

```diff
@@ -0,0 +1,3388 @@
+/* automatically generated by rust-bindgen 0.64.0 */
+
+pub const __GNUC_VA_LIST: i32 = 1;
```

Comment:
You should tweak the bindgen usage so that stuff like this doesn't show up.

Reply:
Will do, do you have any hints how? I actually produced all the pregenerated bindings by building libsqlite3-sys with `buildtime_bindgen`, and then taking the bindings straight from the target/release/build/ directory (:

Reply:
Well, it's not clear to me why this isn't triggering: https://github.com/rusqlite/rusqlite/blob/master/libsqlite3-sys/build.rs#L592

Reply:
See the upgrade.sh scripts for how we usually do this.

Reply:
Just reran bindings generation with `LIBSQLITE3_SYS_BUNDLING` env variable set, the diff looks like it worked exactly as it should: https://github.com/rusqlite/rusqlite/pull/1283/commits/3a7d0b6988b7bcc1550fbd447fde62d07799051d